drivers/display/display_st7701.h

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright (c) 2025 Arduino SA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef ZEPHYR_DRIVERS_DISPLAY_DISPLAY_ST7701_H_
+#define ZEPHYR_DRIVERS_DISPLAY_DISPLAY_ST7701_H_
+
+/**
+ * @name General parameters.
+ * @{
+ */
```

Comment:
Doxygen comments does not make any sense here.

Reply:
Not [addressed](https://github.com/zephyrproject-rtos/zephyr/pull/85569/files#diff-c976fbb76b254b112958bff1a5f1a884af7ed165f9ec48f2f8967b69e6419267R38-R52). In general, marking a reviewer's comments as resolved does not help PR progress.

Reply:
@jfischer-no What do you mean by not addressed? Please provide actionable feedback for the code authors. You highlighted a code section indicating it "doesn't make any sense", which arguably is already a bit vague, then @iabdalkader removed it, so it was actually fine to mark the comment as resolved in this case.
Are you now nitpicking on the fact it's `/**` and not `/*` comments?

Reply:
@kartben There is nothing vague in "Doxygen comments does not make any sense here". Using doxygen comments for local definitions that are not and will never be included in the documentation build does not make any sense. Also, our comment style in C files is clearly documented. You know that, so why are you calling me out here?